skyportal/models.py

```diff
@@ -2517,6 +2517,12 @@ class Spectrum(Base):
         doc="Median UTC ISO time stamp of the exposure or exposures in which the Spectrum was acquired.",
     )
     origin = sa.Column(sa.String, nullable=True, doc="Origin of the spectrum.")
+    type = sa.Column(
+        sa.String,
+        nullable=False,
+        default='source',
+        doc="Type of spectrum: source, host, etc. ",
```

Comment:
```suggestion
        doc="Type of spectrum: source, host, etc.",
```

Reply:
I tried applying the migration but it fails on something unrelated to do with photometry owner_id...

Reply:
Nevermind I managed to get it working. 
The spectra from the demo data get type=source after migration.

Reply:
How long did it take to migrate spectra locally? We have 4,360 spectra in the prod db -- how long would it take to update them?

Reply:
I only have one or two spectra locally, so I really don't know. But I think adding a few thousand strings should not take long under any reasonable conditions...